.circleci/config.yml

```diff
@@ -154,18 +162,36 @@ commands:
           paths:
             - ".local-*"
 
-  prepare-python:
+  prepare-pip-cache:
     parameters:
       py-version:
         <<: *py-version-template
     steps:
-      - attach-and-link:
-          py-version: << parameters.py-version >>
-      - conditional-skip
-      - config-path
       - restore_cache:
+          keys:
+          - pip-cache-{{ arch }}-<< parameters.py-version >>
```

Comment:
I'm assuming that `pip`'s cache is well-behaved, in the sense that it will detect packages that are arch dependent and download the correct package if necessary.

Reply:
Yes, wheels have platform markers in their name if they include platform specific binaries.